Translation corrections (#132)
07th-mod/higurashi-console-arcs: Update/some_012.txt

//rおかーさんはむきかけのりんごを手にしたまま、私に振り向きもしない。おとーさんも私に気づかないのか、それとも無視しているのか、悄然とした表情で画面を見つめていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おかーさんはむきかけのりんごを手にしたまま、私に振り向きもしない。おとーさんも私に気づかないのか、それとも無視しているのか、悄然とした表情で画面を見つめていた。",

> 			NULL, "Mom was hold an unpeeled apple, not looking my way. Dad watched the TV with a crestfallen look on face. Either he didn't notice me either or was just ignoring me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mom was holding an unpeeled apple, not looking my way. And Dad was watching the TV with a crestfallen look on face. Either he didn't notice me either, or he was just ignoring me.", Line_Normal);
```

//rおかーさんは小首を奇妙にかしげながら、私ににたり…、と笑ってみせた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おかーさんは小首を奇妙にかしげながら、私ににたり…、と笑ってみせた。",

> 			NULL, "Mom cocked her head with confusion as she looked at me... and laughed.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mom cocked her head in confusion as she looked at me... and laughed.", Line_Normal);
```

	ClearMessage();

//r朝からずっと続いていた、おばーちゃんの念仏じみた呪唱があまりにも煩くて耳障りだったので、
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "朝からずっと続いていた、おばーちゃんの念仏じみた呪唱があまりにも煩くて耳障りだったので、",
			NULL, "Grandma's ritualistic praying since this morning had been getting on Mom's nerves.", Line_Normal);
	ClearMessage();

//rおかーさんがそれを止めろと怒鳴り、おばーちゃんもそれに対して口汚く罵った結果、…勢いあまって、こうなってしまったそうだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おかーさんがそれを止めろと怒鳴り、おばーちゃんもそれに対して口汚く罵った結果、…勢いあまって、こうなってしまったそうだ。",
			NULL, "So she yelled at her to stop it, but Grandma said some harsh things in response... They must have gotten carried away and it came to this.", Line_Normal);
	ClearMessage();

//夏美の母rvS04/00/310600099.『…仕方ないでしょう？kvS04/00/310600100.　このままだと、私たち一家が雛見沢の人間だってご近所にも知られてしまうしね…。kvS04/00/310600101.夏美だって、おばーちゃんの奇行のせいでみんなから変な目で見られるのは、…嫌でしょう？』
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>夏美の母</color>", NULL, "<color=#f5e6d3>Natsumi's mom</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310600099", 256, TRUE);
	OutputLine(NULL, "『…仕方ないでしょう？",

> 			NULL, "\"...There was nothing to be done about it.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...There was no choice.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>夏美の母</color>", NULL, "<color=#f5e6d3>Natsumi's mom</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310600109", 256, TRUE);
	OutputLine(NULL, "「…だけど、家族の中に裏切り者がいた。",

> 			NULL, "\"...But there wasa  traitor in our family.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...But there was a traitor in our family.", Line_WaitForInput);
```
